facebook-commerce-pixel-event.php

```diff
@@ -34,6 +34,22 @@ public function pixel_base_code() {
 document,'script','https://connect.facebook.net/en_US/fbevents.js');
 %s
 fbq('track', 'PageView', %s);
+
+<!-- Support AJAX add to cart -->
+if(typeof jQuery != 'undefined') {
+  jQuery(document).ready(function($){
+    jQuery( 'body' ).on( 'added_to_cart', function( event ) {
```

Comment:
Same comment here, could you remove the spaces around `'body'` and `event` as well as in front of `'added_to_cart`?